src/System.Net.Mail/src/System.Net.Mail.csproj

```diff
@@ -6,7 +6,6 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <IsPartialFacadeAssembly Condition="'$(TargetGroup)'=='net463'">true</IsPartialFacadeAssembly>
     <NuGetTargetMoniker Condition="'$(TargetGroup)' == ''">.NETStandard,Version=v1.7</NuGetTargetMoniker>
-    <DefineConstants Condition="'$(TargetGroup)' == ''">NETSTANDARD1_7</DefineConstants>
```

Comment:
No, we don't need this constant at all for Mail, as we Mail doesn't use any APIs that are specific to only a particular version of netstandard.